lib/goo/sparql/solutions_mapper.rb

```diff
@@ -135,10 +142,10 @@ def init_unloaded_attributes(found, list_attributes)
       end
 
       def get_value_object(id, objects_new, object, list_attributes, predicate)
-        object = object.object if object && !(object.is_a? RDF::URI)
+        object = object.object if object.is_a?(RDF::Literal)
```

Comment:
the existent condition was better.

because this will raise an exception if object is nil. And I'm not sure that. `!object.is_a(RDF::URI) == object.is_a?(RDF::Literal)`

Reply:
I thought only the literal object had a value ( object.{value or object } , if we're not sure, will always introduce the unnecessary conditions